scripts/setup_spm_test_app_measurement.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Point SPM CI to the tip of main of https://github.com/google/GoogleAppMeasurement
+# so that the release process can defer publish the GoogleAppMeasurement tag
+# until after testing.
+
+sed -i '' 's#exact("[0-9.]*#branch("main#' Package.swift
```

Comment:
An example of input and output would be really helpful for future us.

Reply:
Good idea. Done!